app/js/components/labOrderEntry/LabEntryForm.jsx

```diff
@@ -293,8 +292,9 @@ export class LabEntryForm extends PureComponent {
               <div className="lab-form-wrapper">
                 <div className="lab-category">
                   <ul>
-                    {orderables.map(orderable => (
-                      <li key={shortid.generate()}>
+                    {orderables.map((orderable, index) => (
+                      // eslint-disable-next-line
+                      <li key={index}>
```

Comment:
Instead of using the array `index` for `key` which I think react is not happy about, I believe the `orderable` should have an `id` that can be accessed through `orderable.uuid`, how about you use that one instead of `index`?